.github/release-drafter.yml

```
@@ -28,15 +24,12 @@ template: |
   Special thanks to the following contributors who helped with this release: $CONTRIBUTORS
 
 autolabeler:
-  - label: 'Maven'
-    files:
-      - 'maven-java/*'
+  - label: 'java-sdk'
+    title:
+      - '/.*\(.*java.*\):/i'
```

I believe using the file locations for labelling works well (that what Alpakka does). The title regex can complement that.
```suggestion
  - label: 'java-sdk'
    title:
      - '/.*\(.*java.*\):/i'
   files:
      - 'sdk/java-sdk/*'
```

---

> I believe using the file locations for labelling works well (that what Alpakka does).

It works, but it will create duplicates that might not be desired. Let's say I add a feature for Scala SDK, but cleanup a typo in the java-sdk, then that feature will show up in both categories. Let's try to use conventional commits and give the author of the commit the power to decide.

---

That's just a matter of removing the label in the UI.